blockchain-tests-skip.yml

```diff
@@ -444,9 +444,11 @@ filename:
     - TestStoreGasPrices.json # ef-test #139
     - TestContractInteraction.json # ef-test #140
     - RecursiveCreateContractsCreate4Contracts.json # ef-test #141
+  stTransactionTest:
+    - HighGasPrice.json
```

Comment:
This test fails due to a panic error when loading the file. 
I think the error occurred before the test case was executed, so skipping the file won't have any impact (to be confirmed).
We can therefore remove this file from the list of files to be skipped.